src/controllers/userController.js

```diff
@@ -0,0 +1,81 @@
+const jwt = require("jsonwebtoken")
+const { checkEmptyValues, generateToken, throwError } = require("../utils");
+const { userServicve } = require("../services");
+const { detailUpdateUser } = userServicve;
+
+const detailSignUp = async (req, res) => {
+
+  const accessToken = req.headers.authorization;
+  const { id } = jwt.verify(accessToken, process.env.SECRET);
+  const userId = id;
+
+  if(!accessToken) throwError("INVALID_TOKEN")
+  
```

Comment:
```suggestion

  const accessToken = req.headers.authorization;
  if(!accessToken) throwError("INVALID_TOKEN")
  
  const { id } = jwt.verify(accessToken, process.env.SECRET);
  const userId = id;


  
```
- accessToken이 없는 경우에 대한 예외처리는 token을 verify 하기 전에 미리 와야 합니다.
- accessToken이 없으면, token을 verify하는 과정이 아예 의미가 없기 때문입니다